Misc/Positions_Area Positions 1 UK.txt

```diff
@@ -62,9 +62,12 @@ STC_M_CTR:London Control:199.998:SM:L:SCO:CTR:-:-::
 STC-M_CTR:London Control:199.998:SM:L:SCO-M:CTR:-:-::
 
 ;Military
-EGVV_CTR:Swanwick Military:135.150:VV:M:EGVV:CTR:-:-:6401:6457:N059.52.20.000:W001.16.27.000:N056.30.20.000:W006.52.55.000:N053.24.11.000:W003.09.12.000
-EGQQ_CTR:Scottish Military:134.300:QQ:M:EGQQ:CTR:-:-:4610:4667:N059.52.20.000:W001.16.27.000:N056.30.20.000:W006.52.55.000:N053.24.11.000:W003.09.12.000
-BLKDG_CTR:Blackdog:133.320:BKDG:M:BLKDG:CTR:-:-:6501:6577:N059.52.20.000:W001.16.27.000:N056.30.20.000:W006.52.55.000:N053.24.11.000:W003.09.12.000
-BLKDG_APP:Blackdog:133.320:BKDG:M:BLKDG:APP:-:-:6501:6577:N055.00.00.000:E001.00.00.000:N059.52.20.000:W001.16.27.000:N057.00.00.000:W003.00.00.000:N051.13.25.000:W003.49.06.000
+EGVV_CTR:Swanwick Mil:135.150:VV:M:EGVV:CTR:-:-:6401:6457:N059.52.20.000:W001.16.27.000:N056.30.20.000:W006.52.55.000:N053.24.11.000:W003.09.12.000
+EGVV_N_CTR:Swanwick Mil:134.300:VVN:M:EGVV:CTR:-:-:6401:6457:N059.52.20.000:W001.16.27.000:N056.30.20.000:W006.52.55.000:N053.24.11.000:W003.09.12.000
+EGVV_W_CTR:Swanwick Mil:133.900:VVW:M:EGVV:CTR:-:-:6401:6457:N059.52.20.000:W001.16.27.000:N056.30.20.000:W006.52.55.000:N053.24.11.000:W003.09.12.000
+EGVV_E_CTR:Swanwick Mil:133.320:VVE:M:EGVV:CTR:-:-:6401:6457:N059.52.20.000:W001.16.27.000:N056.30.20.000:W006.52.55.000:N053.24.11.000:W003.09.12.000
```

Comment:
vis points?

Reply:
Modified, designed to work with vis ranges of:
E: 200
W: 250
N: 350